### ISSUE TITLE
Web Site 404 Errors

### ISSUE THREAD

**Issue by KitchM** [DNSSEC-Tools/DNSSEC-Tools#2]
The links on the web site are fraught with 404 errors.  I can't get anywhere, any information or any files.

**Comment by hardaker** [comment]
Sorry, we are in the process of converting the website to github and it's um "less than trivial".  Which links do see that are the problem at the moment?

I just converted a bunch of the wiki links, but many are still likely broken.  I need to run a full http validation tool on the site but haven't yet.

**Comment by KitchM** [comment]
I'll bet it's a big problem.  Glad you know its going on.  How about just posting a warning on the main page?

I was wanting to know all about DNSSEC.  Your's appears to be a major source of information.

I'll look forward to it getting in order, as you can.  Thanks for your efforts.

**Comment by hardaker** [comment]
Sorry, about that; I think posting a warning is a great idea.  I'll go do that now, thanks.

**Comment by KitchM** [comment]
No, thank you.  I appreciate the efforts.  I look forward to learning how to enable DNSSEC on my servers.
